pkg/kubelet/dockertools/docker_manager.go

```diff
@@ -660,6 +660,19 @@ func (dm *DockerManager) runContainer(
 		SecurityOpt: securityOpts,
 	}
 
+	// Set sysctls if requested
+	sysctls, err := api.SysctlsFromPodAnnotation(pod.Annotations[api.SysctlsPodAnnotationKey])
```

Comment:
FYI: @yujuhong Notice that a new feature is being added to docker shim via pod annotations.